### DISCUSSION TITLE
In the programming world, the acronym SaaS stands for "software as a" WHAT?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#358] [Galaxy Trivia] [answered]
**Hint:** 💅

**Comment by peterbaccaro** [comment]
SaaS stands for "software as a service." It is a software delivery model in which software applications are provided over the internet as a subscription-based service.

**Comment by thuongtruong1009** [comment, marked as answer]
In the programming world, the acronym SaaS stands for "software as a service".
